src/models/Service.ts

```diff
@@ -59,6 +59,8 @@ export default class Service {
 
   @observable isBadgeEnabled: boolean = true;
 
+  @observable isMediaBadgeEnabled: boolean = false;
```

Comment:
Could the value in this line be obtained from the default settings in the `config.ts` file?

Reply:
yes we can, will do it.